extensions/amp-sidebar/1.0/toolbar.js

```diff
@@ -47,6 +50,9 @@ export class Toolbar {
     /** @private {Element|undefined} */
     this.targetElement_ = undefined;
 
+    /** @private {Element|undefined} */
+    this.placeholder_ = undefined;
```

Comment:
Sounds good 👍 

I'll rename as I totally forgot to add the placeholder attribute anyways